routes/web.php

```diff
@@ -1,18 +1,11 @@
 <?php
 
+use App\Http\Controllers\CityController;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider and all of them will
-| be assigned to the "web" middleware group. Make something great!
-|
-*/
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/cities', [CityController::class, 'index']);
```

Comment:
For cases like these routes, you should definitely make use of [route groups](https://laravel.com/docs/10.x/routing#route-groups). In this case, you could group by [controller](https://laravel.com/docs/10.x/routing#route-group-controllers), [prefix](https://laravel.com/docs/10.x/routing#route-group-prefixes), and [name](https://laravel.com/docs/10.x/routing#route-group-name-prefixes).